awx/main/dispatch/control.py

```diff
@@ -14,18 +14,18 @@ class Control(object):
     services = ('dispatcher', 'callback_receiver')
     result = None
 
-    def __init__(self, service):
+    def __init__(self, service, host=None):
```

Comment:
All of this is just me refactoring because I didn't like the old way of passing in the hostname.